### DISCUSSION TITLE
Book Cover and Favicon

### DISCUSSION THREAD

**Discussion by SuprimDevkota** [SusheelThapa/command#50] [General]
![2](https://github.com/SusheelThapa/command/assets/80959795/f30fc29a-fa37-4904-9bcb-1b03501dcaab)
![cover1-min](https://github.com/SusheelThapa/command/assets/80959795/92df2ddf-7e16-464c-bec6-81bdf4085423)

**Comment by SusheelThapa** [comment]
@SuprimDevkota  Can you make favicon and Book cover have same logo
